src/core/shape/vertex.js

```diff
@@ -612,18 +613,22 @@ p5.prototype.endContour = function() {
  * @alt
  * Triangle line shape with smallest interior angle on bottom and upside-down L.
  */
-p5.prototype.endShape = function(mode) {
+p5.prototype.endShape = function(mode, count = 1) {
   p5._validateParameters('endShape', arguments);
   if (this._renderer.isP3D) {
     this._renderer.endShape(
       mode,
-      isCurve,
-      isBezier,
-      isQuadratic,
-      isContour,
-      shapeKind
+      this.isCurve,
+      this.isBezier,
+      this.isQuadratic,
+      this.isContour,
+      this.shapeKind,
+      count
     );
   } else {
+    if (count !== 1) {
```

Comment:
I think it could be good to catch if the user tries to use a count < 1 and either log an error or silently fall back to just using a count of 1

Reply:
I was thinking of letting the WebGL implementation do the work since a count less than or equal to 0 could be valid, but prioritizing code safety does make more sense. I'll fix that in a new commit.

Reply:
It's been added to the front of the function below the parameter check in the latest commit.